docs/supportability-metrics.md

```diff
@@ -6,6 +6,78 @@ Supportability metrics are intended to be used to give internal value through an
 ## How
 A timeslice metric is harvested to the JSE/XHR consumer. An aggregation service called Angler aggregates metrics against known labels once per hour and reports a new event with the aggregation findings to a dedicated account.
 
+### WebSockets
+<!--- WebSocket new class was made before the page loaded, ms from page origin --->
+* WebSocket/New/PreLoad/Ms
+<!--- WebSocket new class was made before the page loaded, ms from class init --->
+* WebSocket/New/Preload/MsSinceClassInit
+<!--- WebSocket new class was made after the page loaded, ms from page origin --->
+* WebSocket/New/PostLoad/Ms
+<!--- WebSocket new class was made after the page loaded, ms from class init --->
+* WebSocket/New/PostLoad/MsSinceClassInit
+
+<!--- WebSocket send method was called before the page loaded, ms from page origin --->
+* WebSocket/Send/PreLoad/Ms
+<!--- WebSocket send method was called before the page loaded, ms from class init --->
+* WebSocket/Send/Preload/MsSinceClassInit
+<!--- WebSocket send method was called before the page loaded, bytes from args --->
+* WebSocket/Send/Preload/Bytes
+<!--- WebSocket send method was called after the page loaded, ms from page origin --->
+* WebSocket/Send/PostLoad/Ms
+<!--- WebSocket send method was called after the page loaded, ms from class init --->
+* WebSocket/Send/PostLoad/MsSinceClassInit
+<!--- WebSocket send method was called after the page loaded, bytes from args --->
+* WebSocket/Send/PostLoad/Bytes
+
+<!--- WebSocket close method was called before the page loaded, ms from page origin --->
+* WebSocket/Close-Method/PreLoad/Ms
+<!--- WebSocket close method was called before the page loaded, ms from class init --->
+* WebSocket/Close-Method/Preload/MsSinceClassInit
+<!--- WebSocket close method was called after the page loaded, ms from page origin --->
+* WebSocket/Close-Method/PostLoad/Ms
+<!--- WebSocket close method was called after the page loaded, ms from class init --->
+* WebSocket/Close-Method/PostLoad/MsSinceClassInit
+  
+<!--- WebSocket close event was received before the page loaded, ms from page origin --->
+* WebSocket/Close-Event/PreLoad/Ms
+<!--- WebSocket close event was received before the page loaded, ms from class init --->
+* WebSocket/Close-Event/Preload/MsSinceClassInit
+<!--- WebSocket close event was received after the page loaded, ms from page origin --->
+* WebSocket/Close-Event/PostLoad/Ms
+<!--- WebSocket close event was received after the page loaded, ms from class init --->
+* WebSocket/Close-Event/PostLoad/MsSinceClassInit
```

Comment:
I wonder if we could have a single set of close metric and track what caused the close.